src/pages/IndividualReport.vue

```diff
@@ -235,6 +235,31 @@ const exportToPdf = async () => {
     (exportLoading.value = false);
 };
 
+const optionalAssessments = computed(() =>
+  administrationData.value.assessments.filter((assessment) => assessment.optional),
+);
```

Comment:
I believe we want to grab the optional status from the assignment doc rather than the administration doc because the administration defined conditions that might make the assessment optional. Those conditions are then evaluated in the cloud function that syncs up assignments and the result of that evaluation is written as a boolean to the assignment doc.

So we will likely have to add another `useQuery` to grab the assignment at `/users/${props.userId}/assignments/${props.administrationId}`. Then this `optionalAssessments` can be computed from `assignmentData` rather than `administrationData`.